lib/sparrow/h2_worker.ex

```diff
@@ -451,11 +508,29 @@ defmodule Sparrow.H2Worker do
   defp try_start_conn(state = %State{config: config}, try_count, delay_stream) do
     case start_conn(config) do
       {:ok, new_state} ->
+        :telemetry.execute(
+          [:sparrow, :h2_worker, :conn_success],
+          %{
+            count: try_count,
+            timer: state.restart_connection_timer
+          },
+          extract_worker_info(state)
+        )
+
         %State{new_state | restart_connection_timer: nil}
 
       {:error, reason} ->
         {:ok, delay} = Enum.fetch(delay_stream, try_count)
 
+        :telemetry.execute(
+          [:sparrow, :h2_worker, :conn_fail],
+          %{
+            count: try_count,
+            timer: state.restart_connection_timer
```

Comment:
Isn't this a reference to a timer? If so, this is not actual numeric value. Not only that, this may not even be usable value in near future.